textBasedAdventure.js

```diff
@@ -1,5 +1,62 @@
 const readline = require('readline-sync')
 
-let nameInput = readline.question("Enter your name: ")
-
-console.log(`Hello ${nameInput}!  Welcome to my game.`)
+const play = () => {
+    let userInput;
+    while(userInput !== "yes") {
+        userInput = readline.question("Would you like to play my game?")
+    } 
+    console.log("You've made a good choice")
+    nameInput = readline.question("What's your name traveller?")
+    console.log(`Hello ${nameInput}!  Welcome to my game.`)
+    ageInput = readline.question(`How old are you ${nameInput}?`)
+    if(ageInput < 18) {
+        console.log("Much too young to be traveling by yourself.")
+    } else {
+        console.log("Just the right age for a journey!")
+    }
+    userInput = readline.question("Are you going to the woods or the swamp?")
+    if(userInput === "woods") {
+        woods();
+    } else {
+        swamp();
+    }
+}
+const swamp = () => {
+    console.clear();
+    console.log("The bog witch is before you, approach or run?")
+    let userInput = readline.question("")
```

Comment:
instead of adding the console log on line 26 you can just add that text to the userinput on line 27